pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-paket.php

```diff
@@ -549,6 +550,23 @@ public function crossborder_and_domestic_fields( $dhl_label_items, $is_disabled
 				)
 			);
 
+			$order             = wc_get_order( $order_id );
+			$base_country_code = PR_DHL()->get_base_country();
+
+			if ( ( $base_country_code == 'DE' ) && ( $this->is_shipping_domestic( $order_id ) ) ) {
+				woocommerce_wp_checkbox(
+					array(
+						'id'                => 'pr_dhl_go_green_plus',
+						'label'             => esc_html__( 'GoGreen Plus: ', 'dhl-for-woocommerce' ),
+						'placeholder'       => '',
+						'description'       => '',
+						'value'             => isset( $dhl_label_items['pr_dhl_go_green_plus'] ) ? $dhl_label_items['pr_dhl_go_green_plus'] : $this->shipping_dhl_settings['dhl_default_go_green_plus'],
+						'custom_attributes' => array( $is_disabled => $is_disabled ),
+					)
+				);
+			}
```

Comment:
<img width="244" alt="image" src="https://github.com/user-attachments/assets/630293e6-cc81-4e35-b3cb-8d73275794d4" />
check the style please